src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -237,6 +356,7 @@ private void LogMetricsFromAuthResult(AuthenticationResult authenticationResult,
 
         private void UpdateTelemetry(Stopwatch sw, ApiEvent apiEvent, AuthenticationResult authenticationResult)
         {
+            sw.Stop();
```

Comment:
Call this in the RunAsync method, so that sw Start and Stop are at the same level and easier to see.